flytepropeller/pkg/controller/nodes/common/utils.go

```diff
@@ -107,6 +127,12 @@ func OffloadLargeLiteral(ctx context.Context, datastore *storage.DataStore, data
 		logger.Errorf(ctx, "Failed to offload literal at location [%s] with error [%s]", dataReference, err)
 		return err
 	}
+	// compute the hash of the literal
+	literalDigest, err := pbhash.ComputeHash(ctx, toBeOffloaded)
```

Comment:
nit: can we move this to the if block under

`if toBeOffloaded.GetHash() == "" {`

so we only compute it when necessary?

Reply:
I wanted to do something before overrwriting the value, moved the value assignment at the end along with this being computed in conditional